composer.json

```diff
@@ -4,13 +4,18 @@
     "type": "phpcodesniffer-standard",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4 || 8.0.* || 8.1.*",
+        "php": "^7.4 || 8.0.* || 8.1.* || 8.2.* || 8.3.*",
         "slevomat/coding-standard": ">=6.3",
         "squizlabs/php_codesniffer": ">=3.5"
     },
     "autoload": {
         "psr-4": {
             "PinnacleCodingStandard\\": "PinnacleCodingStandard"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
```

Comment:
I don't think we need this configuration at the library level.

Reply:
I removed it once. It returned XD

Reply:
Okay, that's fine then.